dev/release/README.md

```diff
@@ -181,7 +181,7 @@ Go to the Test PyPI page of Datafusion, and download
 [all published artifacts](https://test.pypi.org/project/datafusion/#files) under `dist-release/` directory. Then proceed
 uploading them using `twine`:
 
-```py
+```bash
 twine upload --repository pypi dist-release/*
```

Comment:
If we are changing this to bash, then we also need to edit the command:

```suggestion
python3 -m twine upload --repository pypi dist-release/*
```

Reply:
Oh, maybe not ... I am still learning about Python. I did not realize that `pip install` could also install executables